include/vkd3d_vk_includes.h

```diff
@@ -61,5 +68,30 @@ typedef struct D3D12_UAV_INFO
     UINT64 gpuVASize;  
 } D3D12_UAV_INFO;
 
+typedef struct D3D12_LATENCY_RESULTS
```

Comment:
Where is this struct defined? Is it an NV Reflex SDK API?

Reply:
In a way, yes, but in the NVAPI part of the stack: https://github.com/jp7677/dxvk-nvapi/blob/ed4338f69eb40b7158a37a5a5729b6c768dfb953/inc/nvapi.h#L19040-L19063

Reply:
The original definition comes from the nvapi header file:
https://github.com/NVIDIA/nvapi/blob/main/nvapi.h#L17254